flake.nix

```diff
@@ -36,11 +36,6 @@
             inherit inputs pkgs;
             modules = [
               {
-                # Make use of the Starship command prompt when this development environment
-                # is manually activated (via `nix develop --impure`).
-                # See https://starship.rs/ for details on the prompt itself.
-                starship.enable = true;
```

Comment:
I realised that if you're using starship (or any other prompt style) as a system prompt, it will continue to be preserved when making use of [direnv](https://github.com/direnv/direnv). So I found in my use case that specifying it at a project level is no longer necessary for me (and just messes with others' prompts).

So happy to remove it.